disko-install

```diff
@@ -35,6 +39,19 @@ serialiaseArrayToNix() {
   echo "$nixExpr"
 }
 
+serialiaseArrayToNixUnquoted() {
```

Comment:
this seems a bit hacky, maybe we should use json instead?

Reply:
Ahh yes, json is a better way. Will do